### DISCUSSION TITLE
Error 100@63 during proxmox/grafana lxc install

### DISCUSSION THREAD

**Discussion by RIPnRhyme** [tteck/Proxmox#400] [Q&A] [answered]
Upgrading via -

bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/grafana-v4.sh)"

Here is the error - 

![image](https://user-images.githubusercontent.com/8877788/182656301-1c09efce-1dd8-4001-8725-533434bd953e.png)

Happy to help dig through logs/troubleshoot. Bit of a container noob.

**Comment by tteck** [comment]
Network issue?
IPv6

**Reply by RIPnRhyme** [reply]
Hmm, I see what you are saying. Need to figure out how/why it is using that..

<img width="592" alt="image" src="https://user-images.githubusercontent.com/8877788/182660979-69a19786-96b4-4243-a714-03bbadd9208e.png">

**Reply by RIPnRhyme** [reply]
oops, this is for docker but all containers currently are sharing the same network device with the same settings.

**Comment by RIPnRhyme** [comment, marked as answer]
I went through the non-default settings within the script. Used all the default settings. There was not one for ipv6, only ipv4 which was interesting. I did have to choose the nic which could have had an ipv6 address but as above mine is not configured for one.

It worked.. no idea why, but it worked.


![image](https://user-images.githubusercontent.com/8877788/182683971-6c1e0607-ac87-4105-9a97-f3573ea10c75.png)
